src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/BitmapIconTests/UnoSamples_Tests.BitmapIcon.cs

```diff
@@ -14,28 +15,49 @@ namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.BitmapIconTests
 	[TestFixture]
 	public partial class BitmapIcon_Tests : SampleControlUITestBase
 	{
+		private bool ContainsColor(Bitmap bitmap, Rectangle rect, Color color)
+		{
+			for (var x = rect.Left; x < rect.Right; x++)
+			{
+				for (var y = rect.Top; y < rect.Bottom; y++)
+				{
+					var pixel = bitmap.GetPixel(x, y);
+					if ((pixel.A, pixel.R, pixel.G, pixel.B) == (color.A, color.R, color.G, color.B))
+					{
+						return true;
+					}
+				}
+			}
+
+			return false;
+		}
```

Comment:
ImageHelper likely contains this method.

Reply:
@jeromelaban I couldn't find it anywhere.

Reply:
Then let's add it in there then, if nothing matches the behavior you're adding: https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.cs